webrtc/RTCPeerConnection-createDataChannel.html

```diff
@@ -72,57 +81,81 @@
-    7.  Let channel have an [[MaxRetransmits]] internal slot initialized to
+    8.  Let channel have a [[ReadyState]] internal slot initialized to "connecting".
+    9.  Let channel have a [[BufferedAmount]] internal slot initialized to 0.
+    10. Let channel have an [[MaxRetransmits]] internal slot initialized to
         option's maxRetransmits member, if present, otherwise null.
-    8.  Let channel have an [[DataChannelId]] internal slot initialized to
-        option's id member, if present, otherwise null.
-    9.  Let channel have an [[Ordered]] internal slot initialized to option's
+    11. Let channel have an [[Ordered]] internal slot initialized to option's
         ordered member.
-    10. Let channel have an [[Protocol]] internal slot initialized to option's
+    12. Let channel have a [[DataChannelProtocol]] internal slot initialized to option's
         protocol member.
-    11. Let channel have an [[Negotiated]] internal slot initialized to option's
-        negotiated member.
-    12. Let channel have an [[DataChannelPriority]] internal slot initialized
-        to option's priority member.
+    14. Let channel have a [[Negotiated]] internal slot initialized to option's negotiated
+        member.
+    15. Let channel have an [[DataChannelId]] internal slot initialized to option's id
+        member, if it is present and [[Negotiated]] is true, otherwise null.
+    17. Let channel have an [[DataChannelPriority]] internal slot initialized to option's
+        priority member.
+    21. If the [[DataChannelId]] slot is null (due to no ID being passed into
+        createDataChannel, or [[Negotiated]] being false), and the DTLS role of the SCTP
+        transport has already been negotiated, then initialize [[DataChannelId]] to a value
+        generated by the user agent, according to [RTCWEB-DATA-PROTOCOL], and skip
+        to the next step. If no available ID could be generated, or if the value of the
+        [[DataChannelId]] slot is being used by an existing RTCDataChannel, throw an
+        OperationError exception.
+
+        Note
+        If the [[DataChannelId]] slot is null after this step, it will be populated once
+        the DTLS role is determined during the process of setting an RTCSessionDescription.
+    22. If channel is the first RTCDataChannel created on connection, update the
+        negotiation-needed flag for connection.
+
 
   6.2.  RTCDataChannel
 
     A RTCDataChannel, created with createDataChannel or dispatched via a
     RTCDataChannelEvent, MUST initially be in the connecting state
 
+    bufferedAmountLowThreshold
+      [...] The bufferedAmountLowThreshold is initially zero on each new RTCDataChannel,
+      but the application may change its value at any time.
+
     binaryType
-      When a RTCDataChannel object is created, the binaryType attribute MUST
+      [...] When a RTCDataChannel object is created, the binaryType attribute MUST
       be initialized to the string "blob".
```

Comment:
Again here, what value do we get here in copying the spec.
Having test titles refer to part of specs seem more useful for instance.

Reply:
I don't have a strong opinion on that. It was in there before so I updated the comments.